modular_skyrat/modules/modular_reagents/code/debuffs.dm

```diff
@@ -0,0 +1,56 @@
+//*
+//DEBUFFS - status effects that in some way hinder or remove the player's function.
+//*
+
+//MOBILE STASIS - same as regular stasis with the immobilization/hands-blocking traits taken off. Non-modular change has it set to also be checked by IS_IN_STASIS(mob).
+/datum/status_effect/grouped/stasis_mobile
+	id = "stasis"
+	duration = -1
+	tick_interval = 10
+	alert_type = /atom/movable/screen/alert/status_effect/stasis
+	var/last_dead_time
+
+/datum/status_effect/grouped/stasis_mobile/proc/update_time_of_death()
+	if(last_dead_time)
+		var/delta = world.time - last_dead_time
+		var/new_timeofdeath = owner.timeofdeath + delta
```

Comment:
Here again, snake_case please. I'm not going to comment it everywhere because it takes forever to leave comments on mobile.